client/app/scripts/superdesk-archive/directives.js

```diff
@@ -512,6 +512,85 @@
             };
         })
 
+        .directive('sdContentResults', ['$location', 'preferencesService', 'packages', 'tags', 'asset',
```

Comment:
This directive is created temporarily, when the development of monitoring will be finished, content and ingest stuffs will be deleted, also as this directive.